src/Tooltip/_variables.scss

```diff
@@ -7,10 +7,15 @@ $tooltip-color:                     $white !default;
 $tooltip-bg:                        $black !default;
 $tooltip-border-radius:             $border-radius !default;
 $tooltip-opacity:                   .9 !default;
```

Comment:
```suggestion
$tooltip-opacity:                   1 !default;
```

Reply:
After this change, everything else looks good to me

Reply:
Done